gawk

```diff
@@ -0,0 +1 @@
+/nix/store/4z79ipgxqn80ns7mpax25zmb77i3ndfw-gawk-5.1.0
```

Comment:
Spurious change (also the other `gawk*`).

Reply:
accidently commited it. should be fixed now.